features/nationaliteit.feature

```diff
@@ -250,3 +250,42 @@ Functionaliteit: Bepalen van de actuele nationaliteit van een persoon
       | datumIngangGeldigheid.type        | Datum                                                |
       | datumIngangGeldigheid.datum       | 2020-07-27                                           |
       | datumIngangGeldigheid.langFormaat | 27 juli 2020                                         |
+
+    Scenario: persoon heeft meerdere nationaliteiten aangeleverd door RNI deelnemer
```

Comment:
Ik heb niet bij de bespreking hiervoor gezeten, maar waarom wordt er in een nationaliteit feature over RNI getest?